devstack/init.sls

```diff
@@ -12,7 +12,6 @@ openstack-devstack ensure package dependencies:
 openstack-devstack ensure user and group exist:
```

Comment:
Perhaps user management might be moved to its own state file `users.sls`, and extended/included here and in `clean.sls`, in order to keep follow DRY?

Reply:
That's a reasonable suggestion - done!!